website/blog/2023-1-4-backups-on-your-coffee-break.md

```diff
@@ -0,0 +1,81 @@
+---
+slug: backups-on-your-coffee-break
+title: "How to Back Up Your Microsoft 365 Data During Your Coffee Break"
+description: "A quick guide to using Corso for data backups"
+authors: nica
+tags: [corso, microsoft 365, backups]
+date: 2023-1-4
+image: ./images/coffee_break.jpg
+---
+
+![By Kenny Louie from Vancouver, Canada -Coffee break, CC BY 2.0, https://commons.wikimedia.org/w/index.php?curid=24336256](./images/coffee_break.jpg)
+
+It’s 10:00 in the morning, and you need coffee and a snack.
+You know you’re supposed to back up the company’s M365 instance before you leave, but it takes so long! Surely a quick
+break won’t matter.
+
+Wrong! While you were in the break room,
+your organization was hit with a malware attack that wiped out many critical files and spreadsheets in minutes.
+Now your cell phone’s ringing off the hook.
+Slapping your forehead with the palm of your hand, you shout,
+“If only backups were faster and easier!”
+
+<!-- truncate -->
+
+Regular backups are increasingly important and must be prioritized; even over your coffee break. A recent study by
+[Arlington Research](https://www.businesswire.com/news/home/20210511005132/en/An-Alarming-85-of-Organizations-Using-Microsoft-365-Have-Suffered-Email-Data-Breaches-Research-by-Egress-Reveals#:~:text=15%25%20of%20organizations%20using%20Microsoft,data%20in%20error%20via%20email.)
+found that 85% of organizations using Microsoft 365 suffered email data breaches in the six months prior to May 2021.
+And it’s not just malware that threatens to corrupt data; downtime can have equally devastating impacts.
+[Two out of every five servers](https://www.veeam.com/blog/data-loss-2022.html)
+experienced an outage over the past 12 months.
+Data can also be lost or corrupted during poorly executed migrations or the cancellation of a software license
+or by human error. And once it’s gone, it’s gone, unless you’ve backed it up.
```

Comment:
Add a note (or break up the paragraph) to also talk about the Recycling Bin. Sometimes folks think of it as a backup substitute but (a) ransomware will empty it out and (b) even Microsoft recommends emptying it out regularly - https://learn.microsoft.com/en-us/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits

You might also want to talk about legal holds but they are limited too (13 for system wide holds) and are manual - https://learn.microsoft.com/en-us/office365/servicedescriptions/sharepoint-online-service-description/sharepoint-online-limits#hold-limits

Reply:
@serverless-mom Did you choose to not include this feedback? I don't have a strong opinion but marking something as resolved without comment usually indicates that the feedback was addressed. Otherwise, please do leave a comment if the request was ignored.

Reply:
gah! didn't push my changes.